### PR TITLE
add signin, signout log

### PR DESCRIPTION
**Pull request by ThuBM**

### CODE REVIEW

File: app/controllers/custom_devise/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CustomDevise::SessionsController < Devise::SessionsController
+  def create
+    super
+    Activity.create(action: Settings.action.login, user: current_user)
+  end
+
+  def destroy
+    byebug
```
Comment (vjnhat) [inline app/controllers/custom_devise/sessions_controller.rb]: remove this line

File: app/controllers/custom_devise/sessions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CustomDevise::SessionsController < Devise::SessionsController
+  def create
+    super
+    Activity.create(action: Settings.action.login, user: current_user)
```
Comment (dieunb) [inline app/controllers/custom_devise/sessions_controller.rb]: remove `()`

File: app/controllers/custom_devise/sessions_controller.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  end
+
+  def destroy
+    super
+    Activity.create(action: Settings.action.logout, user: current_user)
```
Comment (dieunb) [inline app/controllers/custom_devise/sessions_controller.rb]: remove `)(`

File: app/views/custom_devise/sessions/new.html.erb [hunk 4]
```diff
@@ -19,12 +19,12 @@
...
+  <% end %>
 
   <div class="actions">
     <%= f.submit "Log in", class: "btn btn-primary col-md-12" %>
   </div>
   <% end %>
 
-  <%= render "devise/shared/links" %>
+  <%= render "custom_devise/shared/links" %>
 </div>
```
Comment (bapboy18) [inline app/views/custom_devise/sessions/new.html.erb]: Enter

**Pull request merged**